tests/std/tests/P1222R4_flat_set/test.cpp

```diff
@@ -132,9 +119,74 @@ void test_constructors() {
     flat_set<int> a{};
     a = {1, 7, 7, 7, 2, 100, -1};

```

Comment:
Previously, this was passing as the `initializer_list` was implicitly converted to `flat_multiset` and `operator=(flat_set&&)` was called here. Now I can make sure this is calling `operator=(initializer_list)`